_books/ion-1-1/src/macros/system_macros.md

```diff
@@ -29,6 +29,24 @@ This is, essentially, the identity function.
 It produces a stream from any number of arguments, concatenating the streams produced by the nested expressions.
 Used to aggregate multiple values or sub-streams to pass to a single argument, or to produce multiple results.
 
+#### `default`
+
+```ion
+(macro default (expr* default_expr*)
+    // If `expr` is empty...
+    (.if_none (%expr)
+        // then expand `default_expr` instead.
+        (%default_expr)
+        // If it wasn't empty, then expand `expr`.
+        (%expr)
+    )
+)
+```
+
+`default` tests `expr` to determine whether it expands to the empty stream.
+If it does not, `default` will produce the expansion of `expr`.
+If it does, `default` will produce the expansion of `default_expr` instead.
+
```

Comment:
🗺️ User testing found that most uses of `if_none` were implementing `default`. Adding `default` is a nice convenience.